spec/classes/nodejs_spec.rb

```diff
@@ -326,25 +322,11 @@
     end
   end
 
-  context 'when run on an unsupported version of Fedora (18)' do
-    let :facts do
-      {
-        osfamily: 'RedHat',
-        operatingsystem: 'Fedora',
-        operatingsystemrelease: '18'
-      }
-    end
-
-    it do
-      expect { catalogue }.to raise_error(Puppet::Error, %r{The nodejs module is not supported on Fedora 18.})
-    end
-  end
-
-  ['5.0', '6.0', '7.0', '25'].each do |operatingsystemrelease|
+  ['6.0', '7.0', '27'].each do |operatingsystemrelease|
```

Comment:
This and a lot below should be in ```on_supported_os``` too. Could we reuse that instead of duplicating facts?

Reply:
Hmm maybe. We also would need to filter CentOS/OracleLinux/Scientific Linux as we only one EL6/7 types tested once.